src/relay/backend/graph_runtime_codegen.cc

```diff
@@ -661,6 +660,8 @@ struct Handler<std::unordered_map<std::string, dmlc::any>> {
         writer->WriteObjectKeyValue(k, dmlc::get<std::vector<std::vector<int64_t>>>(v));
       } else if (SameType<std::vector<std::string>>(v)) {
         writer->WriteObjectKeyValue(k, dmlc::get<std::vector<std::string>>(v));
+      } else if (SameType<std::vector<dmlc::any>>(v)) {
```

Comment:
What does this change do?

Reply:
Thanks for catching on this. This change supports any type of vectors to be serialized and we need a corresponding process in `json_node.h`. I'll push a commit to fix it.